recipe/meta.yaml

```diff
@@ -36,17 +36,18 @@ test:
 
 about:
   home: https://www.freetype.org/
-  license: GPL-2.0-only and LicenseRef-FreeType
+  license: GPL2
+  license_family: GPL2
```

Comment:
I guess if two licenses are present we can use `license_family: Other`

Reply:
Thanks!